packages/bot-web-ui/src/components/dashboard/dashboard-component/info-panel.tsx

```diff
@@ -19,7 +19,7 @@ const InfoPanel = ({ setInfoPanelVisibility }: TInfoPanel) => {
             >
                 <Icon width='1rem' height='1rem' icon='IcCloseIconDbot' />
             </div>
-            {SIDEBAR_INTRO.map((sidebar_item, index) => {
+            {sidebar_intro.map((sidebar_item, index) => {
                 const { label, content } = sidebar_item;
                 return (
                     <div className='db-info-panel__card' key={index}>
```

Comment:
@sandeep-deriv could you please avoid index as a key in the .map()?

Reply:
done ✅